rust/rsc/src/rsc/main.rs

```diff
@@ -38,9 +38,15 @@ struct ServerOptions {
         long
     )]
     database_url: Option<String>,
+
+    #[arg(
+        help = "Launches the cache without an external database",
+        long
+    )]
+    standalone: bool,
```

Comment:
I don't think this is needed, you could have just specified a sqlite database via the database_url instead

edit: the insecure key part does stand out and it does seem prudent to not generate insecure keys in a way that persists which implies we should tie it to in memory databases.